dbt_coves/tasks/generate/templates.py

```diff
@@ -0,0 +1,53 @@
+import glob
+import shutil
+from pathlib import Path
+
+from rich import console
+
+import dbt_coves
+
+from .base import BaseGenerateTask
+
+console = console.Console()
+
+
+class GenerateTemplatesException(Exception):
+    pass
+
+
+class GenerateTemplatesTask(BaseGenerateTask):
```

Comment:
It looks like you're not using any feature from `BaseGenerateTask`. Why not just inherit from `BaseTask`?

Reply:
@ssassi you are right, but I think it should be from `BaseConfiguredTask` as we decided (as I recall) the user must have `.dbt_coves` folder

Let me know if not, as we should create it if missing, using `cwd` or getting `dbt project dir/`

Reply:
The `.dbt_coves` folder should be always optional.